src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSourceCredential.cs

```diff
@@ -124,5 +124,10 @@ public static PackageSourceCredential FromUserInput(string source, string userna
                     string.Format(CultureInfo.CurrentCulture, Resources.UnsupportedEncryptPassword, source), e);
             }
         }
+
+        internal PackageSourceCredential Clone()
+        {
+            return new PackageSourceCredential(Source, Username, PasswordText, IsPasswordClearText);
```

Comment:
This type has many properties which are not being cloned here (`IsOfficial`, `IsMachineWide`, `IsEnabled`, `Description`, `IsPersistable`, `ProtocolVersion`, ...).

Reply:
Are you referring to `PackageSource`?

Reply:
You're right.  My mistake.